### DISCUSSION TITLE
Use RFID to Trigger Spotify on a Google Chromecast

### DISCUSSION THREAD

**Discussion by bimbamborius** [MiczFlor/RPi-Jukebox-RFID#2220] [Q&A]
I’ve read about the „lost“ feature to play Spotify on the Phonie box.

I’m wondering if it would be possible to trigger a home Automation via Google Home that plays Spotify on an Google Chromecast Audio.

An RFID tag could be the trigger to a specific Song, Album, Artist, Playlist, Podcast. Maybe it’s also possible to follow up in a playlist or podcast.

I would be happy to hear what you think.

**Comment by s-martin** [comment]
The version 2 of Phoniebox can do MQTT (not yet implemented in version 3), so your approach would be possible.

But from my point of view that's not really the main use case for Phoniebox, which is supposed to be a music box on their own.

At this time there's a workaround to make Spotify working again on version 2: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1815#issuecomment-1666535983

For version 3 there is a pull request #2164 in the works.

**Reply by bimbamborius** [reply]
Hi thanks for you reply. 
I guess your right about the scope of the phonie box. 
I found a project that is doing what I imagined. Meaning just focusing on the tactical/physical interaction. 

https://github.com/AidenStickney/modern-record-player

I'm curious for version 3 of the phonie box. Thanks for your great work.
